tripy/docs/pre0_user_guides/02-compiler.md

```diff
@@ -52,13 +52,13 @@ fast_geglu(inp).eval()
 In the example above, we assumed `inp` has a static shape of `(1, 2)`.
 Now, let's assume that the shape of `inp` can vary from `(1, 2)` to `(16, 2)`, with `(8, 2)`
 being the shape we'd like to optimize for. To express this constraint to the compiler,
-we can provide the range of shapes to `InputInfo` using `shape=((1, 8, 16), 2)`.
+we can provide the range of shapes to `InputInfo` using `shape=([1, 8, 16], 2)`.
```

Comment:
Is there a reason to change these to square brackets or just a preference?

Reply:
I just wanted to differentiate them more clearly from the outer parentheses indicating the entire shape. In retrospect, maybe it would have been less confusing to require passing `min`/`opt`/`max` shapes separately instead of doing that on a per-dimension basis.